web/server/go.md

```diff
@@ -0,0 +1,498 @@
+# Web Applications in Go
+
+This section is meant to contain of some of the lessons
```

Comment:
Personal opinion that Effective Go has been very helpful for me (and others have confirmed this). Feel free to suggest structure changes though if there's a better learning style.